### ISSUE TITLE
A choice of 50mW TX out

### ISSUE THREAD

**Issue by s53dz** [df8oe/mchf-github#1562]
A question for possible improvement.

Since mcHF is not equipped with a separate low power TX output I wonder if it would be helpful to have the lowest settable power 50mW instead of now 0.5W? It would certainly help me.

As I see it from the source it would require just one-line change in this file:
`drivers/ui/radio_management.c
`approximately at line 905

change
```
    case    PA_LEVEL_0_5W:
        pf_levelscale = 0.316;        // rescale for 10% of 5 watts (0.5 watts)
        break;
```
to
```
    case    PA_LEVEL_0_5W:
        pf_levelscale = 0.1;        // rescale for 1% of 5 watts (0.05 watts)
        break;
```
I suppose it does not meet any integer limits in calculation elsewhere.
(**EDIT**: I did not test it yet!)

73 Bojan S53DZ

**Comment by s53dz** [comment]
Hi,

I can confirm that this simple change of TX output power from 0,5W to 0,05W works.
I have tested in TUNE and CW modes.
This would also call for a beauty change in display to show 50mW.

Test results of a particular setup just for information:
5.5W 
1.82W
0,9W
0,390W --> 0,040W

I have tested this on an OVI40 UI and mcHF RF.

73 Bojan

**Comment by sp9bsl** [comment]
Hi Bojan,
I would advice to add another setting in power change ring, not simply change the current lowest 0.5W setting to another. Current low power is sometimes useable when testing unknown antenna or using amp which requires such level or just simply low distance test or... etc. Just my thought...

Edit: Besides, lowering the power this way causes degradation of signal to suppressed carrier ratio (!), it is better to use simple 10dB attenuator if you need this to feed external amplifier.

73 Slawek

**Comment by s53dz** [comment]
Hi Slawek.
Thanks for your comment.
I have done that, cycling between FULL, 5W, 2W, 1W, 0.5W and 50mW. And a proper power display.
But you are right. This way the carrier always stays at the same level. Which is awful.

EDIT: Yes, 10 dB is ok, I know, but unfortunately it works also on RX.

73 Bojan

**Comment by s53dz** [comment]
Hi,

Just a short report to tell that in the mean time I have solved the issue locally by adding some logic to it.
Doing it quite old-fashion way. 

With PTT/ line I activate a relay to switch between the RX and TX, where in the TX line I have added a 26 dB pi attenuator. This value I need for the attenuation when using 0.5 W output power. It works well for me.

73 Bojan S53DZ

![if_ptt_sw](https://user-images.githubusercontent.com/17504651/47080645-daf71f80-d208-11e8-8abd-d0ee90d08be4.jpg)
